modules/60crypt-ssh/helper/unlock.c

```diff
@@ -155,6 +155,12 @@ int main( int argc, const char ** argv )
 			NULL
 		};
 
+		if (access(path, F_OK) != 0) {
```

Comment:
This isn't sufficient, because a file that exists but doesn't have the execute bit will still refuse to execute. Also, match the style of the rest of the file. (It's an ugly style, but we aren't going to reformat at this point.)

```suggestion
		if( access( path, X_OK ) != 0 ) {
```